stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/state/messages/reaction/ReactionOption.kt

```diff
@@ -5,11 +5,11 @@ import androidx.compose.ui.graphics.painter.Painter
 /**
  * UI representation of reactions.
  *
- * @param drawable - The ImageVector that's shown in the icon.
- * @param isSelected - If the option is selected or not (already reacted with it).
- * @param type - The String representation of the reaction, for the API. Can be any of:
- * ["like", "love", "haha", "wow", "sad"].
- * */
+ * @param drawable The ImageVector that's shown in the icon.
+ * @param isSelected If the option is selected or not (already reacted with it).
+ * @param type The String representation of the reaction, for the API. Can be any of: ["like", "love", "haha", "wow", "sad"].
+ *              TODO TODO TODO: is this still valid?
```

Comment:
@filbabic Can't this be an arbitrary string now?

Reply:
Do you mean the "like" etc strings? Because those can be arbitrary, both on the BE side (for what we provide) and on the customer side (custom reactions).

Reply:
Yup, that was the question. Can we just remove the entire "can be any of" sentence from this comment then?

Reply:
... or do we want to rephrase it and say that these are some of the values we use by default?

Reply:
I think I wanted to put in the "like" etc examples, just so people know a few examples of what it can be.

If we have documentation for that somewhere (e.g. BE docs) then it'd be great to maybe link to that instead. Otherwise, I'd just remove that part as per your 1st comment!